flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/source/StreamingStartingStrategy.java

```diff
@@ -34,6 +34,13 @@ public enum StreamingStartingStrategy {
    */
   INCREMENTAL_FROM_LATEST_SNAPSHOT,
 
+  /**
+   * Start incremental mode from the latest snapshot exclusive.
+   *
+   * <p>If it is an empty map, all future append snapshots should be discovered.
```

Comment:
maybe it is a typo in the Javadoc above too. `empty map` probably should be `empty table`.